analysis/hostMicrobeInteractions/tutorial_hostMicrobeInteractions.m

```diff
@@ -14,31 +13,32 @@
 % a microbe is joined with a mouse host.
 % 
 % We will use the AGORA resource (Magnusdottir et al., Nat Biotechnol. 2017 
-% Jan;35(1):81-89) in this tutorial. Please download AGORA from <https://www.vmh.life/#downloadview 
-% https://www.vmh.life/#downloadview> and place the models into a folder.
+% Jan;35(1):81-89) in this tutorial. AGORA can be downloaded from <https://www.vmh.life/#downloadview 
+% https://www.vmh.life/#downloadview>.
 % 
 % As the host model, the global mouse reconstruction (Sigurdsson et al., 
-% BMC Systems Biology (2010) 4:140) will be used. Please download the mouse reconstruction 
-% from <https://wwwen.uni.lu/content/download/72950/917509/file/Mus_musculus_iSS1393.zip 
+% BMC Systems Biology (2010) 4:140) will be used. The mouse reconstruction
+% can be downloaded from <https://wwwen.uni.lu/content/download/72950/917509/file/Mus_musculus_iSS1393.zip 
 % https://wwwen.uni.lu/content/download/72950/917509/file/Mus_musculus_iSS1393.zip>.
-% 
-% Define the path to the folder where you stored the AGORA models.
-
-modelPath='YOUR_PATH_TO_AGORA/';
 %% 
 % Initialize the COBRA Toolbox.
-
-initCobraToolbox
-%% 
-% Load the mouse reconstruction.
-
-load('iSS1393.mat');
+initCobraToolbox(0);
+% path to where the COBRA Toolbox is located
+global CBTDIR
+cd([CBTDIR filesep 'tutorials' filesep 'analysis' filesep 'hostMicrobeInteractions'])
+
+%% 
+% Download the mouse reconstruction.
+system('curl -O https://wwwen.uni.lu/content/download/72950/917509/file/Mus_musculus_iSS1393.zip')
```

Comment:
Fair point, but as the commands are run after `initCobraToolbox`, we can rely on `curl`.

Reply:
True, but if we at some point (I don't think so, but it happened before when moving from sourceforge to git) turn away from git to something else, it would need less refactoring.